request_logging/middleware.py

```diff
@@ -133,8 +133,14 @@ def _log_multipart(self, body, logging_context):
         Splits multipart body into parts separated by "boundary", then matches each part to BINARY_REGEX
         which searches for existance of "Content-Type" and capture of what type is this part.
         If it is an image or an application replace that content with "(binary data)" string.
+        This function will log "(multipart/form)" if body can't be decoded by utf-8.
         """
-        body_str = body if isinstance(body, str) else body.decode()
+        try:
```

Comment:
Sorry, this is the opposite of what I meant. More like

```
if isinstance(body, bytes):
    self.logger.log("(multipart/form)")
    return
body_str = body if isinstance(body, str) else body.decode()
...
```

Reply:
I tried to refine the code based on the code snippet, but this will make some of the test cases fail. Some `body` which data type is `bytes` still can be decoded by utf-8. 

Here is one of failed test case.

```console
======================================================================
FAIL: test_request_body_logged (tests.LogTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/CalvinWu/.pyenv/versions/django-request-logging/lib/python3.6/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/CalvinWu/Documents/django-request-logging/tests.py", line 60, in test_request_body_logged
    self._assert_logged(mock_log, body)
  File "/Users/CalvinWu/Documents/django-request-logging/tests.py", line 22, in _assert_logged
    self.assertIn( expected_entry, text )
AssertionError: 'some body' not found in "\x1b[36mPOST /somewhere\x1b[0m\x1b[36m{'HTTP_COOKIE': ''}\x1b[0m\x1b[36m(multipart/form)\x1b[0m"
```

Reply:
Ah, fair enough. Thanks!